### PR TITLE
added missing PHPDoc

### PR DESCRIPTION
**Pull request by OskarStark**
I am targeting this branch, because this is BC

## Subject

Some PHPDoc was missing

### CODE REVIEW

File: src/Admin/UrlGeneratorInterface.php [hunk 1]
```diff
@@ -42,6 +42,7 @@ public function setRouteGenerator(RouteGeneratorInterface $routeGenerator);
      *
      * @param string $name
      * @param mixed  $object
+     * @param array  $parameters
```
Comment (greg0ire) [inline src/Admin/UrlGeneratorInterface.php]: I think not including that one was deliberate, since it gives no additional information

Comment (kunicmarko20) [reply src/Admin/UrlGeneratorInterface.php]: Same, no point adding it if there is no comment about the field

Comment (OskarStark) [reply src/Admin/UrlGeneratorInterface.php]: IMO we should remove it completely or if we keep the php doc it should be complete.
The other have no additional information, too...

Comment (kunicmarko20) [reply src/Admin/UrlGeneratorInterface.php]: There is no `mixed` type and also we support php 5.6 so we can't have a `string` type. I don't know, this is more of a developers POV, maybe we should define a standard and use it everywhere? In my case, I use partial comments (eg comment for types you can't do in your code)

Comment (greg0ire) [reply src/Admin/UrlGeneratorInterface.php]: @OskarStark @kunicmarko20 please read https://github.com/sonata-project/dev-kit/issues/345

Comment (kunicmarko20) [reply src/Admin/UrlGeneratorInterface.php]: then we can close this?

Comment (greg0ire) [reply src/Admin/UrlGeneratorInterface.php]: I think so

**Pull request closed without merging**